src/App.js

```diff
@@ -1,14 +1,47 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
+import {
+    Switch,
+    Route,
+    Link
+} from 'react-router-dom';
+
 import './App.css';
+import Home
+    from "./components/Home";
+import Login
+    from "./components/Login";
+import Blogposts
+    from "./components/Blogposts";
 
 function App() {
-  // We houden in de state bij of iemand is "ingelogd" (simpele versie)
-  const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
+    const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
 
   return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
+      <>
+          <nav className="nav-bar">
+              <ul>
+                  <Link to="/">Home</Link>
+                  {isAuthenticated &&
+                  <Link to="/blogposts">Blogposts</Link>
+                  }
+                  {isAuthenticated
+                      ? <button onClick={() => toggleIsAuthenticated(false)}>Uitloggen</button>
+                      : <Link to="/login">Login</Link>
+                  }
+              </ul>
+          </nav>
+          <Switch>
+                <Route exact path="/">
+                    <Home />
+                </Route>
+                <Route exact path="/blogposts">
+                    <Blogposts isAuthenticated={isAuthenticated} />
+                </Route>
+                <Route exact path="/login">
+                    <Login toggleIsAuthenticated={toggleIsAuthenticated} isAuthenticated={isAuthenticated}/>
+                </Route>
+            </Switch>
+          </>
```

Comment:
ik zou hier ook de route van je `Blog` neerzetten. deze heb je nu in `Blogposts` staan.  Daarmee hou je de hele routing op 1 plek.